vllm/engine/output_processor/multi_step.py

```diff
@@ -127,9 +127,11 @@ def process_outputs(self,
     def _process_decode_and_stop(self, seq: Sequence,
                                  sampling_params: SamplingParams) -> None:
         new_char_count = 0
-        if sampling_params.detokenize:
+        if sampling_params.detokenize and self.detokenizer:
             new_char_count = self.detokenizer.decode_sequence_inplace(
                 seq, sampling_params)
+        else:
+            new_char_count = 0
```

Comment:
This isn't needed because it's already initialized to 0 above

Reply:
Thanks for feedback. I removed it.